browser/ui/sidebar/sidebar_utils.cc

```diff
@@ -276,4 +277,24 @@ SidebarService::ShowSidebarOption GetDefaultShowSidebarOption(
   return ShowSidebarOption::kShowNever;
 }
 
+void ActivatePanelItem(content::WebContents* web_contents,
+                       SidebarItem::BuiltInItemType panel_item) {
+  if (!web_contents) {
```

Comment:
In general, we should avoid unreachable branches. We should have a semantic understanding of each path of control flow. How is this reached? If we don't know, then we should not have a conditional. It may even be worth considering passing `WebContents&`

Reply:
This block no longer existed, resolve this now.